driver-core/src/main/com/mongodb/client/model/UpdateManyModel.java

```diff
@@ -57,6 +63,37 @@ public UpdateManyModel(final Bson filter, final Bson update) {
     public UpdateManyModel(final Bson filter, final Bson update, final UpdateOptions options) {
         this.filter = notNull("filter", filter);
         this.update = notNull("update", update);
+        this.updatePipeline = null;
+        this.options = notNull("options", options);
+    }
+
+    /**
+     * Construct a new instance.
+     *
+     * @param filter a document describing the query filter, which may not be null.
+     * @param update a pipeline describing the update, which may not be null. The update to apply must include only update
+     *               operators.
+     * @since 3.11
+     * @mongodb.server.release 4.2
+     */
+    public UpdateManyModel(final Bson filter, final List<? extends BsonValue> update) {
```

Comment:
The type should be `List<? extends Bson>`, as a pipeline must be an array of BSON documents.

Reply:
Done